apps/rush-lib/src/utilities/Npm.ts

```diff
@@ -8,12 +8,13 @@ export class Npm {
   public static publishedVersions(
     packageName: string,
     cwd: string,
-    env: { [key: string]: string | undefined }
+    env: { [key: string]: string | undefined },
+    extraArgs: string = ''
```

Comment:
>string [](start = 15, length = 6)

Can you make this an optional `string[]` arg, or, perhaps better, pass richer data into this function explicitly defining the registry and the auth token, and letting this function decide what to do with it.

Reply:
Switched to optional `string[]`